test/integration/nodebalancers_test.go

```diff
@@ -85,11 +85,15 @@ func setupNodeBalancer(t *testing.T, fixturesYaml string) (*linodego.Client, *li
 	t.Helper()
 	var fixtureTeardown func()
 	client, fixtureTeardown := createTestClient(t, fixturesYaml)
+	firewall, firewallTeardown, err := createFirewall(t, client)
+	defer firewallTeardown()
 	createOpts := linodego.NodeBalancerCreateOptions{
 		Label:              &label,
-		Region:             getRegionsWithCaps(t, client, []string{"NodeBalancers"})[0],
+		Region:             getRegionsWithCaps(t, client, []string{"NodeBalancers"})[6],
```

Comment:
```suggestion
		Region:             getRegionsWithCaps(t, client, []string{"NodeBalancers"})[0],
```
Maybe put 0 here to avoid index out of bounds issue?

Reply:
Ah this is what I meant in the PR comment. If we go this route of having [6], it will pass locally. But if we put [0] it won't pass locally due to regional unavailability.

Reply:
`[0]` will be fine with the production API, and you can modify `getRegionsWithCaps` to make it only return the available region in the dev environment.